packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -3015,6 +3051,8 @@ Future<TimeOfDay?> showTimePicker({
   EntryModeChangeCallback? onEntryModeChanged,
   Offset? anchorPoint,
   Orientation? orientation,
+  final Icon? switchToInputEntryModeIcon,
```

Comment:
Same idea as above for the non-null defaults.

Also, was making these `final` a mistake? The others are not final either, and we do not enforce that arguments in constructors are final, so I think that the `final` keywords can be removed here?

Reply:
I was again inspired by the approach done in `date_picker.dart`, maybe we can also use default arguments there in a new issue 
https://github.com/flutter/flutter/blob/0500af2b5f750925e355266360307ff4e69c0954/packages/flutter/lib/src/material/date_picker.dart#L1152-L1155